src/Host.cpp

```diff
@@ -3533,18 +3533,36 @@ bool Host::resetBackgroundImage(const QString &name)
 // use it WITHOUT loading a file - at least for the TConsole::importMap(...)
 // case that may need to create a map widget before it loads/imports a
 // non-default (last saved map in profile's map directory.
-void Host::createMapper(bool loadDefaultMap)
+void Host::toggleVisibilityOfOrCreateMapper(const bool loadDefaultMap)
```

Comment:
How about `createOrShowMapper`?

Reply:
It also hides it though - I do agree that name is ugly - lemme think....

1. `toggleShownOrCreateMapper`
2. `showHideOrCreateMapper`

Reply:
Hmm, the semantics here mean nothing at all because they mean everything. Should it be refactored?

Reply:
I'm not sure I get what you mean. :confused:

If you mean, "Why was it called *createMapper* when it does more than that?" I suspect it evolved to get to be like that...